paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/ConfirmStripeIntentParamsFactory.kt

```diff
@@ -48,20 +48,39 @@ internal class ConfirmPaymentIntentParamsFactory(
              We read the top-level SFU to know the merchant’s desired save behavior
              We write payment method options SFU to set the customer’s desired save behavior
              */
-            // At this time, paymentMethodOptions card is the only PM that supports setup future usage
-            paymentMethodOptions = PaymentMethodOptionsParams.Card(
-                setupFutureUsage = when (paymentSelection.customerRequestedSave) {
-                    PaymentSelection.CustomerRequestedSave.RequestReuse ->
-                        ConfirmPaymentIntentParams.SetupFutureUsage.OffSession
-                    PaymentSelection.CustomerRequestedSave.RequestNoReuse ->
-                        ConfirmPaymentIntentParams.SetupFutureUsage.Blank
-                    PaymentSelection.CustomerRequestedSave.NoRequest ->
-                        null
-                }.takeIf {
-                    paymentSelection.paymentMethodCreateParams.typeCode ==
-                        PaymentMethod.Type.Card.code
+            // At this time, paymentMethodOptions card and us_bank_account is the only PM that
```

Comment:
Why did this work for the complete flow, but not the multi-step?

Reply:
This worked for complete flow because the intent is being confirmed through `USBankAccountFormViewModel` and we are constructing the intent manually with `setupFutureUsage`. Should we combine these and update the view model to use `ConfirmStripeIntentParamsFactory`?

Reply:
Seems like it is would be good for them to follow the same flow across all paths?

Reply:
Done